src/main/java/eu/enexa/docker/ContainerManagerImpl.java

```diff
@@ -66,25 +73,35 @@ public String startContainer(String image, String containerName, List<AbstractMa
 
             List<Volume> volumes = new ArrayList<>();
             String expIRI="";
+            String ContainerWritablePath ="";
+            String ContainerModuleInstancePath ="";
```

Comment:
The writable path and the module instance path have to be generated based on the context of the start container call.